src/Core/Domain/Order/QueryResult/OrderForViewing.php

```diff
@@ -237,7 +243,8 @@ public function __construct(
         OrderMessagesForViewing $messages,
         OrderPricesForViewing $prices,
         OrderDiscountsForViewing $discounts,
-        OrderSourcesForViewing $sources
+        OrderSourcesForViewing $sources,
+        string $note
```

Comment:
```suggestion
        string $note = ''
```
To avoid BC breaks, this class is not final